docs/api/native-image.md

```diff
@@ -137,15 +137,17 @@ let image = nativeImage.createFromPath('/Users/somebody/images/icon.png')
 console.log(image)
 ```
 
-### `nativeImage.createFromBuffer(buffer[, scaleFactor])`
+### `nativeImage.createFromBuffer(buffer[width, height, scaleFactor])`
 
 * `buffer` [Buffer][buffer]
+* `width` Integer (optional)
```

Comment:
This argument ordering makes this a breaking change.  I.e. Users currently using `scaleFactor` will have to change their function call to make it keep working.

The new argument order should be `buffer, scaleFactor, width, height` which then means it remains backwards compatible.